base/server/CMakeLists.txt

```diff
@@ -118,6 +118,7 @@ add_custom_command(
 
 if(JAVA_VERSION GREATER 10)
     add_custom_target(pki-server-java11plus-lib ALL
```

Comment:
@edewata This target never executes. In particular, the links `common/lib/jaxb-api.jar` don't ever seem to get created according to the [build log](https://pipelines.actions.githubusercontent.com/bmG54JoT9T4XjcmeUGdblFiDgrq6NGpbGGR1QtrV6gjRWh3H1P/_apis/pipelines/1/runs/960/signedlogcontent/5?urlExpires=2020-09-16T15%3A16%3A37.2496997Z&urlSigningMethod=HMACV1&urlSignature=GiolloZk0L7Ls0kIySfuNJoWgQZOJR09FiNY1Y8cpCA%3D).

`JAVA_VERSION` gets set to 11.

Reply:
Turns out it was because we used `%{java_version}` in our spec instead of `${java_version}`. Ah well :-)